cmds/vboot/vboot.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"crypto/sha1"
+	"crypto/sha256"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/zaolin/go-tpm/tpm"
+	"golang.org/x/crypto/ed25519"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	tpmDevice  string = "/dev/tpm0"
+	mountPath  string = "/mnt/vboot"
+	filesystem string = "ext3"
+)
+
+func dieHard() {
+	if e1, e2, err := syscall.Syscall6(syscall.SYS_REBOOT, syscall.LINUX_REBOOT_MAGIC1, syscall.LINUX_REBOOT_MAGIC2, syscall.LINUX_REBOOT_CMD_POWER_OFF, 0, 0, 0); err != 0 {
```

Comment:
This needs to be in an _linux.go file, e.g. diehard_linux.go. It's very specific to Linux.